### ISSUE TITLE
求 servicebus  edge to  cloud 文档

### ISSUE THREAD

**Issue by shenguangfu0906** [kubeedge/kubeedge#4661]
能把servicebus cloud to edge 文档贴出来吗  不要贴官网了 官网没有  官网只有 cloud  to  edge的

**Comment by RyanZhaoXB** [comment]
Hello, you want the document of router from servicebus to rest, right?  i will propose it later.

**Comment by shenguangfu0906** [comment]
> Hello, you want the document of router from servicebus to rest, right? i will propose it later.

yes ，you know  .there is  only rest to servicebus document on  kubeedge.io!  but  now  i need  send message  from edge to cloud  by service.  i don`t want to use  mqtt.  i think  the  http is better

**Comment by RyanZhaoXB** [comment]
> > Hello, you want the document of router from servicebus to rest, right? i will propose it later.
> 
> yes ，you know .there is only rest to servicebus document on kubeedge.io! but now i need send message from edge to cloud by service. i don`t want to use mqtt. i think the http is better

OK I see. I will make an experiment in my own cluster and write the document.

**Comment by shenguangfu0906** [comment]
> > > Hello, you want the document of router from servicebus to rest, right? i will propose it later.
> > 
> > 
> > yes ，you know .there is only rest to servicebus document on kubeedge.io! but now i need send message from edge to cloud by service. i don`t want to use mqtt. i think the http is better
> 
> OK I see. I will make an experiment in my own cluster and write the document.

thank  you  very much   .

**Comment by shenguangfu0906** [comment]
> > > > Hello, you want the document of router from servicebus to rest, right? i will propose it later.
> > > 
> > > 
> > > yes ，you know .there is only rest to servicebus document on kubeedge.io! but now i need send message from edge to cloud by service. i don`t want to use mqtt. i think the http is better
> > 
> > 
> > OK I see. I will make an experiment in my own cluster and write the document.
> 
> thank you very much .

hi ? what time  will be  push the servicebus  edgetocloud on document?

**Comment by RyanZhaoXB** [comment]
im still working on it. please wait for a few days.

**Comment by shenguangfu0906** [comment]
> im still working on it. please wait for a few days
ok .thanks

**Comment by shenguangfu0906** [comment]
文档什么时候能出来啊

**Comment by Piwriw** [comment]
> im still working on it. please wait for a few days.

Hi，I suggest you to ask the question in the meeting(Europe Time: Wednesdays at 16:30-17:30 Beijing Time (biweekly, starting from Feb. 19th 2020). ([Convert to your timezone.](https://www.thetimezoneconverter.com/?t=16%3A30&tz=GMT%2B8&)). As i know, they are redesigning website. Maybe it will spend some time.

**Comment by a516664625** [comment]
@RyanZhaoXB Hello, is there any documentation available for ServiceBus regarding edge to cloud communication?

**Comment by sgf0906ggy** [comment]
@RyanZhaoXB

**Comment by sgf0906ggy** [comment]
@RyanZhaoXB 
you said a few days. But , It's been five months .
